### PR TITLE
Update README.md

### LINKED ISSUE #1: Create Dockerfile for OpenStreetMap website
The first Dockerfile we should have is for the main OpenStreetMap website.

It needs to install all dependencies, checkout the code, and run the website. It will also need a postgres container to talk to.

Prior art:
  - PR to add Docker to the openstreetmap-website code-base: https://github.com/openstreetmap/openstreetmap-website/pull/1290
 - Dockerfile for openstreetmap-website from 2014: https://github.com/geo-data/openstreetmap-website-docker

cc @geohacker

### PR DESCRIPTION
**Pull request by Andygol**
## What I am changing

fix typo

## How I did it

-

## How you can test it

-

## Related Issues

-

### CODE REVIEW

**Comment by Rub21** [comment]
Looks good, thank you @Andygol !!

File: images/replication-job/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ### Delta replications job container
 
-This contain is responsible for creating the delta replication files it may be set up by minute, hour, etc. Those replications files will be uploaded to a repository in AWs of Gooble Storage, depends on what you are using.
+This contain is responsible for creating the delta replication files it may be set up by minute, hour, etc. Those replications files will be uploaded to a repository in AWS of Google Storage, depends on what you are using.
```
Comment (batpad) [inline images/replication-job/README.md]: Noticed another typo while reviewing this :D - that should be `uploaded to a repository in AWS **or** Google Storage` . cc @Rub21 can do as a separate PR.

**Comment by batpad** [comment]
Thank you so much for these @Andygol !

@Rub21 could you give it a quick look and approve / merge if it looks fine to you? Thanks!

File: images/replication-job/README.md (hunk 1 above)
Comment (Andygol) [reply images/replication-job/README.md]: @batpad tnx for finding.
